slices/fontconfig-config.yaml

```diff
@@ -0,0 +1,32 @@
+package: fontconfig-config
+
+slices:
+  config:
+    essential:
+      - fonts-dejavu-core_fonts
```

Comment:
`fontconfig-config` depends on either of those:
- [fonts-dejavu-core](https://packages.ubuntu.com/noble/fonts-dejavu-core)
- [fonts-liberation](https://packages.ubuntu.com/noble/fonts-liberation)
- [fonts-croscore](https://packages.ubuntu.com/noble/fonts-croscore)
- [fonts-freefont-otf](https://packages.ubuntu.com/noble/fonts-freefont-otf)
- [fonts-freefont-ttf](https://packages.ubuntu.com/noble/fonts-freefont-ttf)
- [fonts-urw-base35](https://packages.ubuntu.com/noble/fonts-urw-base35)
- [fonts-texgyre](https://packages.ubuntu.com/noble/fonts-texgyre)
- [fonts-noto-core](https://packages.ubuntu.com/noble/fonts-noto-core)

I picked the first one, let me know if it is preferred to proceed differently.

Note: `apt install fontconfig-config` defaults to selecting only `fonts-dejavu-core`too:

```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  fonts-dejavu-core fonts-dejavu-mono
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core fonts-dejavu-mono
0 upgraded, 3 newly installed, 0 to remove
```

Reply:
I was wondering about this too. I tested on an `amd64` machine. Is it the same for other architectures too?

Reply:
Did you manage to find out whether it is the same for other architectures?

Reply:
It seems to be yes, I tested `aarch64` and `riscv64` and got the same behavior:
<details>
<summary>aarch64</summary>

```
root@043e60857b97:/# uname -m
aarch64
root@043e60857b97:/# apt install fontconfig-config
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  fonts-dejavu-core fonts-dejavu-mono
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core fonts-dejavu-mono
0 upgraded, 3 newly installed, 0 to remove and 29 not upgraded.
```

</details>

<details>
<summary>riscv64</summary>

```
root@f7530cb8bb1c:/# uname -m
riscv64
root@f7530cb8bb1c:/# apt install fontconfig-config
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  fonts-dejavu-core fonts-dejavu-mono
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core fonts-dejavu-mono
0 upgraded, 3 newly installed, 0 to remove and 28 not upgraded.
```

</details>